common/ip-packet-requests/src/lib.rs

```diff
@@ -1,8 +1,17 @@
+use serde::{Deserialize, Serialize};
+use std::net::{Ipv4Addr, Ipv6Addr};
+
 pub mod codec;
 pub mod request;
 pub mod response;
 
-pub const CURRENT_VERSION: u8 = 3;
+pub const CURRENT_VERSION: u8 = 4;
```

Comment:
Imo it'd be nice to have a short comment here showing what changed between versions (just for our own future reference), like
version3: initial version
version4: ipv6 support

oh and btw. I'm just curious: since we bumped up the version by adding the extra feature, would we still be able to handle v3 requests? if not, imo it's still fine given we're not at the point of needing backwards compatibility

Reply:
I've added the comment, it sounds useful for posterity.
No, there won't be any backwards compatibility, I haven't approached that since it wasn't really needed now. We'll have to make sure to redeploy all gateways.